### PR TITLE
feat(components/datetime): tokenize timepicker styles

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2911/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2911/storybooks/ag-grid-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2911/storybooks/datetime-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2911/storybooks/lists-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2911/storybooks/tabs-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2911/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2911/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2911/playground/)

File: libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss [hunk 1]
```diff
@@ -1,68 +1,198 @@
...
+          var(--sky-space-inset-pillarbox-2_3-bottom-m)
+        )
+        var(
+          --sky-override-timepicker-button-padding-horizontal,
+          var(--sky-space-inset-pillarbox-2_3-left-m)
+        );
+      width: 100%;
+      height: 100%;
+
+      &:focus {
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss]: Should this be `focus-visible`? We don't normally just use `focus`.

Also, we often have these as `focus-visible:not(:active)`. Should we have the `:not(:active)` here too?

Comment (Blackbaud-TrevorBurch) [reply libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss]: Though some of this applied to default that used to use `focus` and some was in a `focus-within` only in modern. 🤔It is odd we aren't using `focus-visible` in modern like we do basically everywhere else - but we normally also didn't have `focus` styles for default. Thoughts?

File: libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss [hunk 2]
```diff
@@ -75,26 +205,37 @@
...
-  button {
-    background-color: $sky-color-white;
-    border-width: 0;
-    padding: $sky-padding-plus-half;
-    padding-top: $sky-padding-half;
-    padding-bottom: $sky-padding-half;
-    width: 100%;
-    height: 100%;
-  }
+  padding: var(
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss:208]: The padding here and around the footer does seem to match the tokens specified in the spec. However, there appears to be more space between the top columns and the button in the footer than in the picture in the spec. Could we check with design on if there is actually supposed to be that much space - or if the bottom side of the top columns maybe isn't supposed to have the content's normal padding?

Comment (Blackbaud-ErikaMcVey) [reply libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss:208]: for this--which mock are you referring to? I only saw where current modern was displayed, I didn't see a mock of v2

Comment (Blackbaud-TrevorBurch) [reply libs/components/datetime/src/lib/modules/timepicker/timepicker.component.scss:208]: Chatted offline - we are leaving this for now

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**